### DISCUSSION TITLE
Navbar drop-down menus

### DISCUSSION THREAD

**Discussion by SamGreenwood1** [google/docsy#1792] [Q&A]
Is there a way to put dropdown menus on the main navbar?

**Comment by deining** [comment]
> Is there a way to put dropdown menus on the main navbar?

Thanks for reaching out. There has been quite some discussion on this topic already, see

* https://github.com/google/docsy/issues/1673
* https://github.com/google/docsy/issues/311

Hope this gets you started!

**Reply by SamGreenwood1** [reply]
So basically I have to modify the theme files to make it work?

**Comment by BethProedrou** [comment]
These links do not really answer the question. If I need a dropdown menu for the versions and a dropdown menu for something else, then the suggested solutions do not work.
